Expensify/App
# Hide transfer balance option when balance is zero

### Details
Check that balance is zero before displaying the button.

### Fixed Issues

$ [GH_LINK](https://github.com/Expensify/App/issues/7601)

### Tests / QA
1. Log into account with a $0 wallet balance
2. Verify that the `transfer balance` button does not appear

### Tested On


### Screenshots


#### Web
<img width="376" alt="Screen Shot 2022-02-11 at 2 17 31 PM" src="https://user-images.githubusercontent.com/24466196/153678595-941b4268-fc05-4ba7-b55f-1eb1f556d22c.png">
